data/org.gnome.shell.gschema.xml.in

```diff
@@ -144,6 +144,14 @@
       <summary>Keybinding that pauses and resumes all running tweens, for debugging purposes</summary>
       <description></description>
     </key>
+
```

Comment:
The message of this commit should reflect the list of fixes that went in after the initial commit (so e.g. the gnome-system-monitor part is not true anymore)